src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -376,6 +376,31 @@ public override bool CanWrite
         }
     }
 
+    ///<Summary>
+    ///  Converter for Description property in the NodeModel class.
+    ///</Summary>
+    public class DescriptionConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return (objectType == typeof(String));
+        }
+
+        /// When deserializing, we do not want to read this property from the file
+        /// so null is being returned. This is to convert the Description property
+        /// to the localized language. 
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            return null;
```

Comment:
If possible, can we use `string.empty` instead?

Reply:
@QilongTang If I am returning empty string instead of null value, it is overwriting the node's description to an empty string and that is being deserialized. The value from node's config is not being picked up.

Reply:
Thanks! Good to know!